src/main/resources/test-data.xml

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+    <include file="changelog-master.xml" relativeToChangelogFile="true"/>
+
+    <!--insert dac 1-->
+    <changeSet author="rfricke" id="test">
+        <validCheckSum>ANY</validCheckSum>
```

Comment:
I don't think you'll need any of these lines since we're always working off of a fresh database. These are required more for the case when you need to update an existing changeset file that's already been run and exists in the changelog table. That table stores a checksum of the changeset and usually throws an error if the checksums are different. This drops that constraint and ignores modified changeset files.